WalletWasabi.Fluent/Models/UI/ApplicationSettings.cs

```diff
@@ -43,10 +43,14 @@ public partial class ApplicationSettings : ReactiveObject
 	[AutoNotify] private string _localBitcoinCoreDataDir;
 	[AutoNotify] private bool _stopLocalBitcoinCoreOnShutdown;
 	[AutoNotify] private string _bitcoinP2PEndPoint;
+	[AutoNotify] private string _dustThreshold;
+
+	// Coordinator
 	[AutoNotify] private string _coordinatorUri;
 	[AutoNotify] private string _maxCoordinationFeeRate;
 	[AutoNotify] private string _maxCoinJoinMiningFeeRate;
-	[AutoNotify] private string _dustThreshold;
+	[AutoNotify] private string _absoluteMinInputCount;
+	[AutoNotify] private string _readMoreUri;
```

Comment:
this was added by mistake?

Reply:
L53

Reply:
Thanks!! This is the exact moment I realized "wait..... `ReadMoreUri` should not be configureable. I forgot to remove it and didn't notice the leftover